git/foundations_git/git_basics.md

```diff
@@ -199,11 +199,3 @@ The following questions are an opportunity to reflect on key topics in this less
 - [How do you commit the files in the staging area and add a descriptive message?](#git-commit)
 - [How do you push your changes to your repository on GitHub?](#git-push)
 - [How do you look at the history of your previous commits?](#git-log)
-
-### Additional resources
-
-This section contains helpful links to related content. It isn't required, so consider it supplemental.
```

Comment:
We'll want to keep this section with the bullet below added as we outline in our [lesson layout guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lesson-layout)

```
- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```